distribution/changelog.txt

```diff
@@ -9,6 +9,7 @@
 - Feature: [#13509] [Plugin] Add ability to format strings using OpenRCT2 string framework.
 - Feature: [#13512] [Plugin] Add item separators to list view.
 - Feature: [#13583] Add allowed_hosts to plugin section of config.
+- Feature: [#13587] Enhanced track designer with ability to add/remove scenery and footpaths.
```

Comment:
and allows for up to 128 ride types to be loaded. or something along those lines.

Reply:
Perhaps create two entries, as this PR adds a feature (scenery) and an improvement (more rides).